WORKFLOW.md

```diff
@@ -0,0 +1,100 @@
+# Development Workflow
+
+## Overview ✌🏻
+
+This guide provides information about:
+
+* Cleaning and setup
+* New sprints, iterations or particular features
+* Prepare code base releases
+* Update working branches after release
+* Add fixes or hot-fixes on release and master branches
+
+## Workspace 🧽
+
+Before start to work on an issue as a developer you need to ensure that your local repository is up to date.
+
+First, is necessary to update your *develop* branch to match remote changes.
+
+```shell
+git checkout develop
+```
+
+```shell
+git fetch origin
+```
+
+```shell
+git pull origin develop
+```
+
+Finally, you must clean remote references to branches:
+
+```shell
+git remote prune origin
+```
+
+Also, do not forget to delete any local branch that doest not exists on the remote any more:
+
+```shell
+git branch -d <branch-name>
+```
+
+For more information about the last command, we recommend to check [this thread](http://stackoverflow.com/questions/4040717/git-remote-prune-didnt-show-as-many-pruned-branches-as-i-expected).
+
+## Work 💪🏻
+
+### Analyze
+
+After getting issues assigned, to work on an issue you must review the issue assigned to you, understand it and ask if there are any questions.
+
+* If there is any question, you need to write a comment on the issue thread mentioning the current issue administrator or reporter.
+* Wait for the clarifications.
+
+### Branch naming 
+
+#### Types
+
+In general the first part of your branch name should match the type of task your working on followed by the `/` character:
+
+* chore
+* feature
+* fix
+* refactor
+* hotfix
+
+Examples:
+
+* `chore/gradle-dependencies`
+* `feature/recurrent-loans-view`
+* `fix/login-flow`
+* `refactor/calculator-base`
+* `hotfix-1.0.1/user-login-redirection`
+
+### Actually coding 
+
+When you are ready to work on the issue, you need to create a new branch. 
+
+As a developer you need to create a local branch using the nomenclature defined at the beginning of this section and push it to the remote repository. *It is important that the branch starts from the develop remote reference.*
+
+```shell
+git fetch origin && git checkout develop
+```
+
+```shell
+git checkout -b <branch-name>
+```
+
+```shell
+git push origin <branch-name>
+```
+
+After the working branch is ready, you must make the proper changes and push your to work to the remote repository.
+
+### Request a review
```

Comment:
In the entire file, you refered to the readers or the developers as `you` but in this section you referred to them as 'the developer'. it's not consistent, but maybe it's just me 😅.

`You must take the proper changes and push your work`
`The developer must take the proper changes and push his work`

Reply:
Nice catch! I think we could keep the `you` approach